libs/model/src/policies/handlers/notifyAddressOwnershipTransferred.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  EventHandler,
+  notificationsProvider,
+  WorkflowKeys,
+} from '@hicommonwealth/core';
+import { ZodBoolean } from 'zod';
+import { models } from '../../database';
+
+export const notifyAddressOwnershipTransferred: EventHandler<
+  'AddressOwnershipTransferred',
+  ZodBoolean
+> = async ({ payload }) => {
+  // TODO: should we ignore when old_user_email is null?
```

Comment:
We can still send the old user an in-app notification. Maybe get product weigh in here - @dillchen